artifacts/definitions/Windows/Applications/Firefox/Downloads.yaml

```diff
@@ -0,0 +1,67 @@
+name: Custom.Windows.Application.Firefox.Downloads
```

Comment:
Should remove the Custom prefix from the built in set

Reply:
Thanks for the prompt. Fixed in e229847